src/SMU-Keithley_6430/main.py

```
@@ -178,8 +237,86 @@ def configure(self) -> None:
+            self.port.write(f":SOURce:{source_type}:STARt {self.list_start}")
+            self.port.write(f":SOURce:{source_type}:STOP {self.list_end}")
+            self.wait_for_complete()
+
+            if self.list_step_points_type == "Step width:":
+                self.port.write(f":SOURce:{source_type}:STEP {self.list_step_points_value}")
+            else:  # Linear or logarithmic points
+                spacing = "LINear" if self.list_step_points_type == "Points (lin.):" else "LOGarithmic"
+                self.port.write(f":SOURce:SWEep:SPACing {spacing}")
+                self.port.write(f":SOURce:SWEep:POINts {self.list_step_points_value}")
+
+            number_of_points = int(self.port.query(f":SOURce:SWEep:POINts?"))
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)